tools/backup_db/restore.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-docker run --rm -i --network fleet_default mysql:5.7 bash -c 'gzip -kdc - | mysql -hmysql -uroot -ptoor fleet'  < backup.sql.gz
+docker run --rm -i --network fleet2_default mysql:5.7 bash -c 'gzip -kdc - | mysql -hmysql -uroot -ptoor fleet'  < backup.sql.gz
```

Comment:
Let's leave out this change. Btw if you change the directory name back to `fleet` you'll not need this change in order to run the restore script.